torch/nn/parallel/distributed.py

```diff
@@ -539,7 +539,15 @@ def __init__(
         super(DistributedDataParallel, self).__init__()
         Joinable.__init__(self)
         self.logger = None
-        if not any((p.requires_grad for p in module.parameters())):
+
+        if hasattr(module, "_ddp_params_and_buffers_to_ignore"):
+            self.parameters_to_ignore = module._ddp_params_and_buffers_to_ignore
```

Comment:
nit: maybe call it `_params_buffers_to_ignore` to align what this actually mean?

Reply:
@wanchaol there are a couple of internal usages that seem to check against self.parameters_to_ignore (or explicitly set this), maybe we can leave this part the same to not break BC?